activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-rest/src/main/java/org/activiti/cloud/services/modeling/rest/controller/ModelController.java

```diff
@@ -157,7 +159,7 @@ public void updateModelContent(
         if (file.getSize() > maxModelFileSize) {
             throw new FileSizeException("File size exceeded");
         }
-        modelService.updateModelContent(findModelById(modelId), multipartToFileContent(file));
+        modelService.updateModelContent(findModelById(modelId), multipartToFileContent(file), new HashMap<>());
```

Comment:
since the method handles null, maybe null can be used instead of instantiating an empty map?

Reply:
It makes sense. Updated.